config/routes.rb

```diff
@@ -6,7 +6,7 @@
   get "/pulls" => "pull_requests#index", :as => :pulls #routes to list of pulls
   get "/users" => "users#index", :as => :users #routes to list of users
   get "/user/:id" => "users#show", :as => :user #routes to specific user's contributions
-  get "/repos" => "repos#dashboard", :as => :repos #routes to list of repos and stats
+  get "/repos" => "repos#index", :as => :repos #routes to list of repos and stats
```

Comment:
Consider updating the comment to note that his route is to list repos for the search filter